README.md

```diff
@@ -63,6 +63,17 @@ repository.
 If you encounter any issues whilst using cert-manager, and your issue is not
 documented, please [file an issue](https://github.com/jetstack/cert-manager/issues).
 
+Possible resources for *help*:
+
+* The official channel `#cert-manager` on `kubernetes.slack.com`
+
+> There is also a good chance to get some support on non-official support
+> channels for *cert-manager*, but be aware that these are rather general
+> kubernetes discussion channels.
+
+* Slack channels like `#kubernetes-users` or `#kubernetes-novice` on `kubernetes.slack.com`
+* If you think you've run into a bug, and you cannot find an existing issue open that describes your problem, please file an issue against the repo [here](https://github.com/jetstack/cert-manager/issues).
```

Comment:
Just noticed we already have this link above the block you're adding. Don't need to repeat it here 😄